src/jack.py

```diff
@@ -56,10 +56,29 @@
 NETSTARTING = _lib.JackTransportNetStarting
 """Waiting for sync ready on the network."""
 
+PROPERTY_CREATED = _lib.PropertyCreated
+"""A property was created.  See `Client.set_property_change_callback()`."""
+PROPERTY_CHANGED = _lib.PropertyChanged
+"""A property was changed.  See `Client.set_property_change_callback()`."""
+PROPERTY_DELETED = _lib.PropertyDeleted
+"""A property was deleted.  See `Client.set_property_change_callback()`."""
+
 _SUCCESS = 0
 _FAILURE = 1
 
 
+def _decode(cdata):
+    return _ffi.string(cdata).decode()
```

Comment:
This handles only ASCII strings, doesn't it?

Maybe allow to pass an encoding?

```
def _decode(cdata, encoding='utf-8'):
    return _ffi.string(cdata).decode(encoding)
```

Reply:
I was assuming Python 3, where `encoding="utf-8", errors="strict"` is the default.

I guess for Python 2 support we would have to make this explicit. But I don't think we have to make it configurable.

Reply:
The default can still be changed, even in Python 3, though.